text/0000-const-trait-impls.md

```diff
@@ -0,0 +1,1006 @@
+- Feature Name: `const_trait_methods`
+- Start Date: 2024-12-13
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#67792](https://github.com/rust-lang/rust/issues/67792)
```

Comment:
this should probably be changed to a new tracking issue, see: https://github.com/rust-lang/rust/issues/67792#issuecomment-2603351678